api/hasura/actions/deleteEpoch.ts

```diff
@@ -0,0 +1,52 @@
+import assert from 'assert';
+
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+import { z } from 'zod';
+
+import { authCircleAdminMiddleware } from '../../../api-lib/circleAdmin';
+import { adminClient } from '../../../api-lib/gql/adminClient';
+import {
+  errorResponse,
+  zodParserErrorResponse,
+} from '../../../api-lib/HttpError';
+import {
+  deleteEpochInput,
+  composeHasuraActionRequestBody,
+} from '../../../src/lib/zod';
+
+async function handler(request: VercelRequest, response: VercelResponse) {
+  try {
+    const {
+      input: { payload: input },
+    } = composeHasuraActionRequestBody(deleteEpochInput).parse(request.body);
+
+    const { circle_id, id } = input;
```

Comment:
Don't we wont to verify that the user is an admin on the circle? using custom permissions on the table? If not, do we need to do it here?

Reply:
doesn't the `authCircleAdminMiddleware` cover this part

Reply:
Yes i'm sorry I missed/forgot about that.